compliance/fedramp.md

```diff
@@ -0,0 +1,11 @@
+---
+description: fedramp
+keywords: standards, compliance, security, documentation
```

Comment:
I'm not sure about using the keyword "documentation" in the docs either. WDYT @johndmulhausen